src/components/navigation/NavItems.js

```diff
@@ -63,9 +63,10 @@ const NavItemsWrapper = styled.div`
 const PortalOuterWrapper = styled.div`
   -webkit-overflow-scrolling: touch;
   height: 100%;
-  overflow: scroll;
+  overflow-y: scroll;
   position: fixed;
   right: 0;
+  scroll-behavior: smooth;
```

Comment:
What does this for? It's not very well supported :(

https://caniuse.com/?search=scroll-behavior

Reply:
ya, you're right :( i removed it. it doesn't seem to actually be affecting scroll behavior on desktop.